cpp/ql/test/query-tests/Best Practices/Unused Entities/UnusedStaticVariables/test.cpp

```diff
@@ -7,6 +7,9 @@ static int staticVar4 = staticVar3; // GOOD (used)
 static int staticVar5; // BAD (unused)
 static int staticVar6 = 6; // BAD (unused)
 static __attribute__((__unused__)) int staticVar7; // GOOD (unused but this is expected)
+const int constVar8 = 8; // BAD (const defaults to static)
+extern const int constVar9 = 9; // GOOD
+static int staticVar10 = 10; // GOOD [FALSE POSITIVE] (referenced in a never instantiated template)
```

Comment:
I don't think this would have been caused by _recent_ extractor changes, but it's always good to have test cases that show the limitations of our queries.